management_webpage/flaskr/useCDM.py

```diff
@@ -30,7 +30,10 @@ def getCDMUri():
         
         ## Units are always their own instance (measurement -> unit -> literal), therefore filtering the unit instances.
         FILTER (!(STRSTARTS(str(?variableUri), "http://purl.obolibrary.org/obo/UO_"))).
-        FILTER (?variableUri != sio:SIO_001112).
+        ## This filter will have be modified based on changes in the future.
+        ## Another one might have to be added is schema1:Date depending on the
+        ## role it plays
+        FILTER (?variableUri != sio:SIO_001112 && ?variableUri != sty:T081 && ?variableUri != sty:T079 && ?variableUri != sty:T080).
```

Comment:
You forgot to define the prefix `sty`, so the query doesn't work.

Reply:
Ah, I'm surprised it worked for me. I'll add the prefix and give it a go. Thanks.

Reply:
@aiaragomes Pushed the fix for this issue only.